README.rst

```diff
@@ -126,6 +126,10 @@ The compiled packages will go in ``bin/packages/*/openwisp``.
 Alternatively, you can configure your build interactively with ``make menuconfig``, in this case
 you will need to select the *openwisp-monitoring* variant and *netjson-monitoring* by going to ``Administration > admin > openwisp``:
 
+.. image:: https://raw.githubusercontent.com/openwisp/openwrt-openwisp-monitoring/media/docs/OpenWISP%20Compilation.gif
+   :target: https://github.com/openwisp/openwrt-openwisp-monitoring/media/docs/OpenWISP%20Compilation.gif
+   :alt: OpenWISP Monitoring Package Compilation
```

Comment:
I believe the whole GIF is not relevant here.

I think we should do the following:

- show this GIF at the top of the "Compiling openwrt-openwisp-monitoring" section
- make a shorter version of this GIF with only the make menuconfig steps to be shown here

What do you think @devkapilbansal?

Reply:
:thinking: Other steps are somewhat implicit too. Therefore we can remove that too.

Also, should we remove this section [here](https://github.com/openwisp/openwrt-openwisp-monitoring/blob/master/README.rst#compiling-openwrt-openwisp-monitoring)

```
git clone https://git.openwrt.org/openwrt/openwrt.git
cd openwrt
git checkout openwrt-21.02

# configure feeds
echo "src-git openwisp https://github.com/openwisp/openwisp-monitoring.git" > feeds.conf
cat feeds.conf.default >> feeds.conf
./scripts/feeds update -a
./scripts/feeds install -a
make menuconfig
# go to Administration > admin > openwisp and select the packages you need interactively
make tools/install
make toolchain/install
make package/openwrt-openwisp-monitoring/compile
```

Reply:
I don't understand.

Reply:
Do you refer to the previous discussion to remove it? I think we could just simplify and remove it. The GIF can be put at the beginning of the section.